.gitignore

```diff
@@ -0,0 +1,19 @@
+.svn/
+.hg/
+.hgtags/
+log
+*.py[cod]
+build
+doc/_build
+dist/
+pylint.egg-info/
+.tox
+*.sw[a-z]
+doc/features.rst
+pyve
+build-stamp
+debian/files
```

Comment:
Right now this simply ports the ignored patterns from the .hgignore.  If increased specificity is desired, I think a follow-on ticket/PR for general ignore file improvement is appropriate.